frontend-ssr/components/cli_git/GitCli.module.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../styles/variables';
+
+.commands {
+    width: 35vw;
+
+    .commands__cli {
+        height: 50vh;
+        overflow: auto;
+        padding: 2vh 1vw;
+        @extend %gradientBackground, %blackGradientBackground;
+
+        .command {
+            color:color(blue);
+        }
+        .success {
+            color:color(green);
+        }
+        .error {
+            color: color(red);
+        }
+        p {
+            margin: 0;
+        }
+    }
+}
+.form {
+    position: relative;
+    color: color(white);
+    .form__input {
+        width: 100%;
+        background-color: color(gray);
+        padding: 1em;
+        border: none;
+        &:focus ~ span::after {
+            width: 0;
+        }
+    }
+    span {
+        position: absolute;
+        top: 14px;
+        left: 7px;
+        animation: animate 1s linear infinite; 
+        &::after {
+                content: '';
+                width: 25px;
+                height: 2px;
+                background-color: color(#fff);
+                display: inline-block;
+        }
+    }
+    
+}
+.btn {
+    visibility: hidden;
+}
+ 
+@keyframes animate{ 
```

Comment:
We may need this animation in other parts as well. So let's externalize it. Maybe create an `animations.scss` file and import it from `index.scss`. Also, let's change it's name. `animate` is WAY WAY to generic.